core/comet/service.go

```diff
@@ -0,0 +1,63 @@
+package comet
+
+import (
+	"context"
+	"time"
+)
+
+// Service is an interface that can be used to get information specific to Comet
+type Service interface {
```

Comment:
Can we call this `HeaderService` or `InfoService` or something? There could be other types of comet related services

Reply:
changed to infoservice